Makefile

```diff
@@ -880,6 +883,10 @@ template-name:
 		$(MAKE) -s -C $(SRC_DIR)/linux-template-builder template-name; \
 	done
 
+update-repo-iso-testing: ISO_VERSION=$(shell cat $(SRC_DIR)/$(INSTALLER_COMPONENT)/build/ISO/qubes-x86_64/iso/build_latest)
+update-repo-iso-testing:
```

Comment:
What about just `update-repo-iso`? Or maybe it should be even called `upload-iso`, since it isn't really about any repository?

Reply:
upload-iso sounds good. It misunderstood our talk about '-testing" or not earlier.